src/System.Numerics.Vectors/src/System/Numerics/Vector.cs

```diff
@@ -2752,11 +2752,11 @@ public string ToString(string format, IFormatProvider formatProvider)
         }
 
         /// <summary>
-        /// Returns a boolean indicating whether any single pair of elements in the given vectors are equal.
+        /// Returns a boolean indicating whether each pair of elements in the given vectors are not equal.
```

Comment:
I guess it depends on how you interpret the wording here, but I also find the original wording clearer (although it's saying the exact opposite of what it means 😄).

Can we change it to this?
```
Returns a boolean indicating whether any single pair of elements in the given vectors are not equal.
```

(essentially just adding a "not" before "equal"). Let me know if that wording doesn't make sense for you, @Gregory-Bell